content/en/agent/versions/_index.md

```diff
@@ -18,11 +21,13 @@ Datadog recommends you update Datadog Agent with every minor and patch release,
 <p>
 Upgrading to a major Datadog Agent version and keeping it updated is the only supported way to get the latest Agent functionality and fixes. The Agent has frequent update releases, though, and managing updates at enterprise scale can be challenging. That doesn't mean you should wait for major releases before updating. The right update cadence for your organization will depend on your infrastructure and your configuration management practices, but aim for monthly.</p>
 <p>
-To update the Datadog Agent core between two minor versions on a given host, run <a href="https://app.datadoghq.com/account/settings#agent">the corresponding install command for your platform</a>.</p>
+To update the Datadog Agent core between two minor versions on a given host, run <a href="/agent/versions/upgrade_between_agent_minor_versions">the corresponding install command for your platform</a>.</p>
 <p>
 Datadog Agent release numbering follows <a href="https://semver.org/">SemVer</a> rules.</p>
 </div>
 
+[1]: /agent/versions/upgrade_between_agent_minor_versions
```

Comment:
I don't think you need this relative link.

Reply:
You're right, I forgot to remove this after some experiments. Thanks!